index.html

```diff
@@ -1275,29 +1275,39 @@ <h2>Characters</h2>
-		<p>For example, the Bangla user-perceived character <em>kshī</em> <span class="codepoint" translate="no"><bdi lang="bn">ক্ষী</bdi></span> is composed of four characters: <span class="codepoint" translate="no"><code class="uname">U+0995 BENGALI LETTER KA</code> + <code class="uname">U+09CD BENGALI SIGN VIRAMA</code> + <code class="uname">U+09B7 BENGALI LETTER SSA</code> + <code class="uname">U+09C0 BENGALI VOWEL SIGN II</code>.
-		<p>Unicode splits these into two grapheme clusters, unless language-specific tailoring is applied. For more information, see our article <a href="https://www.w3.org/International/articles/definitions-characters/index.en.html#characters">Character encodings: Essential concepts</a>.</p>
-    </aside>
-
-    <aside class=example title="Examples of graphemes and user-perceived characters">
-		<p>Here is the word for "Unicode" in the Latin, Katakana, Arabic, and Devanagari scripts.</p>
-		<p class=bigtext>Unicode
-		   <span lang=ja>&#x30E6;&#x30CB;&#x30B3;&#x30FC;&#x30C9;</span>
-		   <span lang=ar dir=rtl>&#x064A;&#x0648;&#x0646;&#x064A;&#x0643;&#x0648;</span>
+    <aside class=example title="Examples of graphemes and typographic units">
+		<p>Here is the word for "Unicode" in the English (which uses the Latin script) and Hindi (which uses the Devanagari script).</p>
+		<p class=bigtext>Unicode<br>
 		   <span lang=hi>&#x092F;&#x0942;&#x0928;&#x093F;&#x0915;&#x094B;&#x0921;</span>
 		</p>
-		<p>The last example, in the Devanagari script (which is used to write Hindi, among other languages) is made up of four graphemes:</p>
+		<p>The Hindi word is made up of four graphemes:</p>
 		<p class=bigtext lang=hi>&#x092F;&#x0942;&nbsp;&#x0928;&#x093F;&nbsp;&#x0915;&#x094B;&nbsp;&#x0921;</p>
     </aside>
+    
+	<p>[[Unicode]] provides a definition for approximately computing [=grapheme=] boundaries. The boundaries defined by [[Unicode]] are called [=grapheme clusters=]. Unless otherwise specified, the term [=grapheme cluster=] in this document refers to what [[UAX29]] refers to as an "extended default grapheme cluster".</p>
+	
+    <p>For many languages and scripts, there is little difference between a [=grapheme=] and a [=grapheme cluster=]. However, for some languages and scripts, particularly those found in South Asia, the difference can be important. Work is on-going at Unicode to narrow or eliminate gaps between grapheme and grapheme cluster definitions. For example, this section previously used an example in Bangla, but the Bengali script was corrected in Unicode 15.1.</p>
+
+	<aside class="example" title="Difference between Unicode grapheme cluster and a typographic unit">
+	<p>In Balinese, the word "ᬅᬲ᭄ᬢ", which means "to be", is composed of this character sequence: 
+	<span class="codepoint" translate="no"><bdi lang="ban">&#x1B05;</bdi><code class="uname">U+1B05 BALINESE LETTER AKARA</code></span>
```

Comment:
I think this would be better as a (vertical) list, rather than inline.

Reply:
done